queries.yaml

```diff
@@ -924,3 +951,8 @@ post_process:
         white: [255, 255, 255]
         yellow: [255, 255, 0]
         yellowgreen: [154, 205, 50]
+
+  - fn: vectordatasource.transform.drop_layer
+    params:
+      layer: admin_areas
+      start_zoom: 15
```

Comment:
Should this be dropped for all zooms?

Reply:
Yup, well spotted! Thanks, fixed in #1501.